common/persistence/cassandra/cassandraPersistenceUtil.go

```diff
@@ -995,19 +995,15 @@ func createTransferTasks(
 			targetDomainID = task.(*p.CancelExecutionTask).TargetDomainID
 			targetWorkflowID = task.(*p.CancelExecutionTask).TargetWorkflowID
 			targetRunID = task.(*p.CancelExecutionTask).TargetRunID
-			if targetRunID == "" {
```

Comment:
Why did you remove this?

Reply:
This was just used internally as a default value marker for Cassandra's frozen<transfer_task>.RunID which was a UUID. It would write a well known UUID if the RunID was not specified and then it would swap this back out on deserialization with an empty string. Now that RunID is exposed as a byte array itself, this seems unneeded and we can propagate nil if thats what the customer provides.

Reply:
See func createTransferTaskInfo() for the other side of this